samples/secret.yaml

```diff
@@ -31,9 +31,11 @@
   # Allowed Values: string
   # Default Value: "none"
   nasName: "nas-server"
-- username: "admin"
-  password: "Password123"
-  systemID: "2b11bb111111bb1b"
-  endpoint: "https://127.0.0.2"
-  skipCertificateValidation: true 
-  mdm: "10.0.0.3,10.0.0.4"
+
+# # To add more PowerFlex systems, uncomment the following lines and provide the required values
+# - username: "admin"
+#   password: "Password123"
```

Comment:
Change "Password123" to "password" if we are just using an example.

Reply:
done